### PR TITLE
Add a note to EKS Addon guide about sidecar auto-injection support

### PR DESCRIPTION
**Pull request by levan-m**
### What does this PR do? What is the motivation?

Next addon release includes Operator which support agent sidecar autoinjection for pods schedule on EKS Fargate instances. PR adds a callout on top of the page.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/containers/guide/operator-eks-addon.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ further_reading:
     text: 'Datadog and Kubernetes'
 ---
 
+<div class="alert alert-info"> Starting with v0.1.9 Datadog Operator add-on supports automatic Agent sidecar injection in pods scheduled on Fargate instances. See <a href="https://docs.datadoghq.com/integrations/eks_fargate/?tab=datadogoperator#admission-controller-using-datadog-operator">this guide</a> for more details.
+</div>
```
Comment (hestonhoffman) [inline content/en/containers/guide/operator-eks-addon.md:12]:
```suggestion
<div class="alert alert-info">Starting with v0.1.9, the Datadog Operator add-on supports automatic Agent sidecar injection in pods scheduled on Fargate instances. See <a href="https://docs.datadoghq.com/integrations/eks_fargate/?tab=datadogoperator#admission-controller-using-datadog-operator">this guide</a> for more details.
</div>
```

**Review by hestonhoffman — APPROVED** [review]
Small edit

**Comment by levan-m** [comment]
0.1.9 isn't published yet so please don't merge it.

**Comment by levan-m** [comment]
Addon was published today and this PR can be merged

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**